Bottom: 2235d2b7a6c2f9fc34cc2b2eaa4e7d658b916c99
Top:    dd8cc5630109dfae1a260254cd2607c2844d5d43
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-01-06 16:50:35 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
